src/SUMMARY.md

```diff
@@ -18,6 +18,9 @@
     - [So you want to add a new option to rustc?](./compiler/new_option.md)
     - [Major Change Proposals](./compiler/mcp.md)
     - [Membership](./compiler/membership.md)
+    - [Prioritization](./compiler/prioritization.md)
+      - [Priority Levels](#)
+      - [Prioritization Process](./compiler/prioritization-process.md)
```

Comment:
My proposal is to place this document under the "Priorit..." chapter, under what I see the place for the soon-to-be-merged PR [describing the priority levels](https://github.com/rust-lang/rust-forge/pull/390)

Reply:
:+1:, I'd place the process first and call it just Process